lib/matplotlib/axes/_axes.py

```diff
@@ -1517,33 +1517,12 @@ def plot(self, *args, scalex=True, scaley=True, **kwargs):
 
         A format string consists of a part for color, marker and line::
 
-            fmt = '[color][marker][line]'
+            fmt = '[marker][line][color]'
```

Comment:
Not sure it is worth to make that change. Anyway we should explicitly state that order does not matter.

Reply:
Well, it does in the sense that C11 doesn't mean the same thing as 1C1.

Reply:
I feel that 'b-' is much more used in practical code than '-b'. Maybe also because of the current docs. I wouldn't want to change the habits of users for that. 'C11' really an edge case, and is just now eagerly collecting digits. People can still go for '1C1' in that specific case (or use kwargs), I would not generally recommend switching the order.

Reply:
I'm not recommanding it (or even coming closing to thinking about deprecating the other order); I'm just putting the docs in the order that always works...
At least if by any chance a new user discovers the minilanguage by reading this specific docstring (which likely never happens), they'll have it in the right order.

Reply:
> add a sentence like,

> Note that other combinations like [color][marker][line] are equally supported but might lead to ambiguities concerning colors and markers.

Done.

> Then change the example strings further down the page to match with the new format.

That was already done.